manual/moderncv_userguide.tex

```diff
@@ -759,11 +813,29 @@ \subsection{Adjusting lengths}
 One should be careful though, as the length is normally calculated to avoid any overlap with the personal information.
 This should be used at one's own typographical risk.
 
+The spacing between the hints column and the CV entries can be adjusted by setting \code{\\separatorcolumnwidth}. The default value in units of \code{\\textwidth} is $0.025$.
```

Comment:
```suggestion
The spacing between the hints column and the CV entries can be adjusted by setting\\\code{\\separatorcolumnwidth}. The default value is \code{0.025\\textwidth}.
```
The line exceeds the margin in the PDF, so I added a manual line break.

Reply:
Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)